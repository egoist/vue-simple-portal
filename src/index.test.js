import { mount } from 'vue-test-utils'
import SimplePortal from './'

const $target = document.createElement('div')
$target.id = 'target'
document.body.appendChild($target)

test('it works', () => {
  const wrapper = mount({
    template: '<simple-portal target="target">hi</simple-portal>',
    components: {
      SimplePortal
    }
  })
  expect($target.outerHTML).toMatchSnapshot()
})
