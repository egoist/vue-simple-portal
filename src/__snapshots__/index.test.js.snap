// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `"<div id=\\"target\\"><div>hi</div></div>"`;
